src/collector/gecko-pending.js

```diff
@@ -0,0 +1,119 @@
+import {max} from 'lodash';
+import sli from '../sli';
+import slo from '../slo';
+import eb from '../eb';
+
+const MINUTE = 60 * 1000;
+
+sli.declare({
```

Comment:
I **love** the design of this file... even though I haven't looked the collector logic, slo, sli or eb, yet, I totally feel like I know how this will work and what the end result will be... As well as how to add another sli...